packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/BarGraph/BasicBarGraphs/AxisInfoTextLabels.jsx

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import Carbon from '@cerner/carbon-graphs/lib/js/carbon';
 import utils from '@cerner/carbon-graphs/lib/js/helpers/utils';
-import '@cerner/terra-graphs-docs/lib/Css/ExampleGraphContainer.module.scss';
 import '@cerner/terra-graphs/lib/components/Graph.module.scss';
 import '@cerner/terra-graphs/lib/components/Bar/BarGraph.module.scss';
+import '@cerner/terra-graphs-docs/lib/Css/ExampleGraphContainer.module.scss';
```

Comment:
Does this make any difference I mean in changing the place of import?

Reply:
Yes, during my testing, reordering imports resulted in different snapshots. This is because there are styles in `ExampleGraphContainer.module.scss` that override `Graph.module.scss`. Different ordering of the imports results in different outputs for the snapshots. Putting `ExampleGraphContainer.module.scss`at the end is also consistent with the original react examples we made.